definitions/npm/react-testing-library_v5.x.x/flow_v0.67.1-/test_react-testing-library_v5.x.x.js

```diff
@@ -72,6 +72,7 @@ describe('render', () => {
     container,
     unmount,
     baseElement,
+    asFragment,
```

Comment:
Could you please add a test for it? All/most of the other methods have one.

Reply:
Added!